src/sagemaker/huggingface/model.py

```diff
@@ -534,4 +544,5 @@ def serving_image_uri(
             image_scope="inference",
             base_framework_version=base_framework_version,
             serverless_inference_config=serverless_inference_config,
+            inference_tool=inference_tool,
```

Comment:
This sounds reasonable to me, how did this work for `neuron` before.

Reply:
If no inference_tool, there's a function `sagemaker/image_uris::_get_inference_tool` which returns 'neuron' `if instance_type_family.startswith("inf") or instance_type_family.startswith("trn")`

Reply:
@claytonparnell Thanks for helping on debugging the tests. It looks good from my side.